### PR TITLE
Composer.lock file updated

### PR DESCRIPTION
**Pull request by reshmai**

### CODE REVIEW

**Comment by reshmai** [comment]
Hi,

I got conflicts for content_hash only on develop. is it ok to remove old one and keep mine?

**Comment by melwong-jcc** [comment]
Can go either way. Ideally, we should be regenerating that hash once we merge all changes for deployment but since it's an optional feature, not sure how much reliance we've had with it.

File: composer.lock
Comment (melwong-jcc) [inline composer.lock:23057]: Not sure why this is downgrading for a courtyard update so maybe best to keep at 2.3.0?

**Review by melwong-jcc — APPROVED** [review]

**Comment by melwong-jcc** [comment]
Remember, don't merge develop into your branch to resolve. Pull latest develop, merge your branch into it, resolve conflict, push updated develop.

**Pull request merged**
